src/main/java/es/thalesalv/chatrpg/application/config/DefaultConfiguration.java

```diff
@@ -86,15 +86,15 @@ public void init() throws StreamReadException, DatabindException, IOException {
                     .filter(p -> !personaRepository.existsById(p.getId()))
                     .forEach(persona -> {
                         LOGGER.info("Default persona named {} not in DB. Ingesting it.", persona.getName());
-                        persona.setOwner(bot.getId());
+                        persona.setOwnerDiscordId(bot.getId());
                         personaService.savePersona(persona);
                     });
 
             defaultWorlds.stream()
                     .filter(w -> !worldRepository.existsById(w.getId()))
                     .forEach(world -> {
                         LOGGER.info("Default world named {} not in DB. Ingesting it.", world.getName());
-                        world.setOwner(bot.getId());
+                        world.setOwnerDiscordId(bot.getId());
```

Comment:
Is this right? I would expect `bot.getId` to return the id of the bot, not the owner. Or does the bot own these assets?

Reply:
Yes. These are the default assets, as the name of the class suggests. These are all owned by the bot.